packages/babel-plugin-transform-block-scoping/test/fixtures/general/for-for-x/input.js

```diff
@@ -0,0 +1,17 @@
+function w() {
+    for (let i = 0; i < y; i++) {
+        let outer = {}
+        for (var key in someObj) {
+            let x = () => outer;
+        }
+    }
+}
+
+function w2() {
```

Comment:
What is the difference between these two functions?

Reply:
One is `for in` and the other is `for of`. :)

Reply:
Ohh I played the "find the difference between the two pictures" for like 3 minutes before giging up :grimacing: